x/name/types/codec.go

```diff
@@ -18,11 +17,6 @@ func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 		&MsgCreateRootNameRequest{},
 	)
 
-	registry.RegisterImplementations(
-		(*govtypesv1beta1.Content)(nil),
-		&CreateRootNameProposal{},
```

Comment:
The `CreateRootNameProposal` proto message should be marked as deprecated with a message pointing people to `MsgCreateRootNameRequest`.